test/tools/ossfuzz/config/solidity.dict

```diff
@@ -0,0 +1,210 @@
+"continue"
+"delete "
+"do"
+"else"
+"for (a=0;a<2;a++)"
+"function bid() public payable {"
+"modifier onlySeller() {"
+"if"
+"new "
+"return"
+"switch"
+"throw"
+"try"
+"while"
+"pragma solidity ^90.90.0"
+"pragma experimental ABIEncoderV2;"
+"pragma experimental SMTChecker;"
+"pragma solidity >=0.4.0 <0.6.0;"
+"x << y"
+"x * 2**y"
+"x / 2**y"
+"int x = -2**255;"
+"x % y"
+"0**0"
+"<<"
+">>"
+"|"
+"&"
+"^"
+"~"
+"<="
+"<"
+"=="
+"!="
+">="
+">"
+"+"
+"-"
+"*"
+"/"
+"%"
+"**"
+"fixed "
+"unfixed "
+"ufixed128x128 "
+"fixed128x128 "
+"float "
+"double "
```

Comment:
Do we need float and double? They are not keywords in Solidity.

Reply:
Removing them.